dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -14,1955 +14,6933 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
 */
-
-SET FOREIGN_KEY_CHECKS=0;
+SET
+FOREIGN_KEY_CHECKS = 0;
 
 -- ----------------------------
 -- Table structure for QRTZ_BLOB_TRIGGERS
 -- ----------------------------
-DROP TABLE IF EXISTS `QRTZ_BLOB_TRIGGERS`;
-CREATE TABLE `QRTZ_BLOB_TRIGGERS` (
-  `SCHED_NAME` varchar(120) NOT NULL,
-  `TRIGGER_NAME` varchar(200) NOT NULL,
-  `TRIGGER_GROUP` varchar(200) NOT NULL,
-  `BLOB_DATA` blob,
-  PRIMARY KEY (`SCHED_NAME`,`TRIGGER_NAME`,`TRIGGER_GROUP`),
-  KEY `SCHED_NAME` (`SCHED_NAME`,`TRIGGER_NAME`,`TRIGGER_GROUP`),
-  CONSTRAINT `QRTZ_BLOB_TRIGGERS_ibfk_1` FOREIGN KEY (`SCHED_NAME`, `TRIGGER_NAME`, `TRIGGER_GROUP`) REFERENCES `QRTZ_TRIGGERS` (`SCHED_NAME`, `TRIGGER_NAME`, `TRIGGER_GROUP`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+DROP
+    TABLE
+        IF EXISTS ` QRTZ_BLOB_TRIGGERS `;
+
+CREATE
+    TABLE
+        ` QRTZ_BLOB_TRIGGERS `(
+            ` SCHED_NAME ` VARCHAR(120) NOT NULL,
+            ` TRIGGER_NAME ` VARCHAR(200) NOT NULL,
+            ` TRIGGER_GROUP ` VARCHAR(200) NOT NULL,
+            ` BLOB_DATA ` BLOB,
+            PRIMARY KEY(
+                ` SCHED_NAME `,
+                ` TRIGGER_NAME `,
+                ` TRIGGER_GROUP `
+            ),
+            KEY ` SCHED_NAME `(
+                ` SCHED_NAME `,
+                ` TRIGGER_NAME `,
+                ` TRIGGER_GROUP `
+            ),
+            CONSTRAINT ` QRTZ_BLOB_TRIGGERS_ibfk_1 ` FOREIGN KEY(
+                ` SCHED_NAME `,
+                ` TRIGGER_NAME `,
+                ` TRIGGER_GROUP `
+            ) REFERENCES ` QRTZ_TRIGGERS `(
+                ` SCHED_NAME `,
+                ` TRIGGER_NAME `,
+                ` TRIGGER_GROUP `
+            )
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8;
 
 -- ----------------------------
 -- Records of QRTZ_BLOB_TRIGGERS
 -- ----------------------------
-
 -- ----------------------------
 -- Table structure for QRTZ_CALENDARS
 -- ----------------------------
-DROP TABLE IF EXISTS `QRTZ_CALENDARS`;
-CREATE TABLE `QRTZ_CALENDARS` (
-  `SCHED_NAME` varchar(120) NOT NULL,
-  `CALENDAR_NAME` varchar(200) NOT NULL,
-  `CALENDAR` blob NOT NULL,
-  PRIMARY KEY (`SCHED_NAME`,`CALENDAR_NAME`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+DROP
+    TABLE
+        IF EXISTS ` QRTZ_CALENDARS `;
+
+CREATE
+    TABLE
+        ` QRTZ_CALENDARS `(
+            ` SCHED_NAME ` VARCHAR(120) NOT NULL,
+            ` CALENDAR_NAME ` VARCHAR(200) NOT NULL,
+            ` CALENDAR ` BLOB NOT NULL,
+            PRIMARY KEY(
+                ` SCHED_NAME `,
+                ` CALENDAR_NAME `
+            )
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8;
```

Comment:
This format is really ugly...

Reply:
LOL. This is formatted by [dbeaver](https://github.com/diffplug/spotless/tree/main/plugin-maven#dbeaver), we may choose other formatting options, or switch to use [prettier](https://github.com/diffplug/spotless/tree/main/plugin-maven#prettier)... 🤣

Reply:
Although it looks a bit strange, it helps the SQL code more readable once you get used to it. 🤣

Reply:
I still think this is not as good as before.

Reply:
> I still think this is not as good as before.

LOL I'm looking into prettier to see if we could have a better one.

Reply:
@SbloodyS @kezhenxu94 I've tested [prettier](https://github.com/diffplug/spotless/tree/main/plugin-maven#prettier) with `Spotless`, it seems there's some kind of bug to use `prettier-sql-plugin` with `Spotless`. Thus, `dbeaver` formatter is our only choice with `Spotless` for `SQL` formatting at this moment. We have to trade-off here. If consistency of formatting is more important, we will pick this `dbeaver formatter`. If you think the style is too ugly and prefer not to using it, we just leave those `SQL` scripts unformatted at present.